src/home/views/index.tsx

```diff
@@ -8,57 +9,64 @@ import { orderListUrl } from "../../orders/urls";
 import { productListUrl, productVariantEditUrl } from "../../products/urls";
 import { OrderStatusFilter, StockAvailability } from "../../types/globalTypes";
 import HomePage from "../components/HomePage";
-import { HomePageQuery } from "../queries";
+import { useHomePage } from "../queries";
 
 const HomeSection = () => {
   const navigate = useNavigator();
   const { user } = useUser();
   const { channel } = useAppChannel();
 
+  const noChannel = !channel && typeof channel !== "undefined";
+
+  const { data } = !noChannel
+    ? useHomePage({
```

Comment:
I'm afraid conditionally using hook might cause React errors in the future - https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level